README.md

```diff
@@ -13,6 +13,7 @@ The goal of this tutorial is to demonstrate how to get started with an integrati
 Please see the [Introduction](https://github.com/Watson-Explorer/wex-wdc-integration-samples) for an overview of the integration architecture, and the tools and libraries that need to be installed to create Java-based applications in Bluemix.
 
 - An [IBM Bluemix account](https://bluemix.net/)
+- [Cloud Foundry](https://github.com/cloudfoundry/cli/releases)
```

Comment:
Should use https://github.com/IBM-Watson/wex-appbuilder-samples/blob/master/intro-to-bluemix-for-app-builder.md for introduction to Bluemix for WEX / App Builder.  The old repo is basically abandoned at this point.